macros/generate_model_yaml.sql

```diff
@@ -17,24 +17,31 @@
     {% do return(model_yaml) %}
 {% endmacro %}
 
-{% macro generate_model_yaml(model_name, upstream_descriptions=False) %}
+{% macro generate_model_yaml(model_names=[], upstream_descriptions=False) %}
 
-{% set model_yaml=[] %}
-{% set column_desc_dict =  codegen.build_dict_column_descriptions(model_name) if upstream_descriptions else {} %}
+    {% set model_yaml=[] %}
 
-{% do model_yaml.append('version: 2') %}
-{% do model_yaml.append('') %}
-{% do model_yaml.append('models:') %}
-{% do model_yaml.append('  - name: ' ~ model_name | lower) %}
-{% do model_yaml.append('    description: ""') %}
-{% do model_yaml.append('    columns:') %}
-
-{% set relation=ref(model_name) %}
-{%- set columns = adapter.get_columns_in_relation(relation) -%}
+    {% do model_yaml.append('version: 2') %}
+    {% do model_yaml.append('') %}
+    {% do model_yaml.append('models:') %}
 
-{% for column in columns %}
-    {% set model_yaml = codegen.generate_column_yaml(column, model_yaml, column_desc_dict) %}
-{% endfor %}
+    {% if model_names is string %}
+        {{ exceptions.raise_compiler_error("The name argument to ref() must be a list, got <class 'string'>: " ~ number) }}
```

Comment:
This is a strange error message: 
- This doesn't seem to be an argument to ref, should that be `model_names`?
- What is `number`? I don't see it defined anywhere
- Saying `<class 'string'>` feels like the sort of thing that comes from an automatically-generated error message. 

Have taken a swing at an alternative, but lmk if I've misunderstood its purpose
```suggestion
        {{ exceptions.raise_compiler_error("The `model_names` argument must always be a list, even if there is only one model.") }}
```

Reply:
Hi again, Joel! thank you for rewriting this, I was following this [exceptions.raise_compiler_error](https://docs.getdbt.com/reference/dbt-jinja-functions/exceptions) example and didn't know how far away from it I could wander. It has been fixed now!